lib/pages/themes-page.js

```diff
@@ -77,7 +90,7 @@ export default class ThemesPage extends BaseContainer {
 		return this.driver.findElement( popOverMenuSelector ).isDisplayed();
 	}
 
-	static _getThemeSelectionXpath(phrase ) {
+	static _getThemeSelectionXpath( phrase ) {
 		return by.xpath( `//h2[text()[contains(.,'${ phrase }')]]/../../../../div[not(contains(concat(' ', @class, ' '), ' is-active '))]//span` );
```

Comment:
is there anyway to simplify this complex mess (my code I think) - maybe by adding a data attribute to the wp-calypso screen?